src/structs.rs

```diff
@@ -393,6 +409,8 @@ pub struct FrameworkOptions<U, E> {
     pub prefix_options: crate::PrefixFrameworkOptions<U, E>,
     /// User IDs which are allowed to use owners_only commands
     pub owners: std::collections::HashSet<serenity::UserId>,
+    /// Internal field to count the number of assigned Command IDs
+    pub id_count: usize,
```

Comment:
Hmm, kinda unfortunate to have an particular method's implementation detail variable in the main framework options... another sign that adding commands should really be done by a separate builder a la `framework_options.commands(|c| c.command(...).command(...).command(...))`. I'll look into that soon hopefully

Reply:
Modified this field to be more useful on fdcd6d1